src/box/lua/net_box.c

```diff
@@ -185,7 +185,6 @@ static const char netbox_request_typename[] = "net.box.request";
  * We keep a reference to each C function that is frequently called with
```

Comment:
Why do you need this patch? It does not look right to terminate the connection when one of triggers for one of public events fails. It is not critical for the connection. It is like terminating the connection when `conn.space.test:insert` fails. Please, lets leave it working as before.

Reply:
It won't terminate the connection if one of the triggers registered for a public event fails, because we don't call the user callbacks directly here. Instead we call a dispatcher function that invokes the callbacks in a fiber, under a pcall, so it can only fail on a memory allocation error so pcall is redundant here.

https://github.com/tarantool/tarantool/blob/e0e6fe106d11855631a2673dbc72a670378669aa/src/box/lua/net_box.lua#L766-L780

https://github.com/tarantool/tarantool/blob/e0e6fe106d11855631a2673dbc72a670378669aa/src/box/lua/net_box.lua#L934-L968